### DISCUSSION TITLE
当一个Edge链接到一个Node时，是否可以用鼠标拉走Edge连接到其他Node上

### DISCUSSION THREAD

**Discussion by Alimjan2013** [antvis/X6#3227] [Q&A] [answered]
从A节点连接到B节点时，是否有办法把已经连到B上的线拉到C上

**Comment by NewByVector** [comment, marked as answer]
使用 [target-arrowhead](http://x6.antv.antgroup.com/api/registry/edge-tool#source-arrowhead-%E5%92%8C-target-arrowhead) 工具。
